corona-warn-app/cwa-app-android
# Family test result change notifications (EXPOSUREAPP-12333)

# How to test?
- Scan pending family tests 
- Then change the result to final positive or negative 
- Home tab should be opened to trigger refresh 
- You should see a notification 
- Notification should open the empty family list screen for now 
## Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12333
